upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -719,6 +719,49 @@ There you can find links to upgrade notes for other versions too.
         + },
         ```
 
+
+- if you want to add stylelint rules to check style coding standards [#1511](https://github.com/shopsys/shopsys/pull/1511)
```

Comment:
The whole doc block shoul be under [`Tools`](http://docs.shopsys.com/en/8.0/contributing/guidelines-for-writing-upgrade/#section-shopsysframework)

Reply:
Whole block moved under Tools section